include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1692,7 +1694,8 @@ int bt_le_ext_adv_update_param(struct bt_le_ext_adv *adv,
  * @brief Delete advertising set.
  *
  * Delete advertising set. This will free up the advertising set and make it
- * possible to create a new advertising set.
+ * possible to create a new advertising set if the limit @kconfig{CONFIG_BT_EXT_ADV_MAX_ADV_SET}
+ * was reached.
```

Comment:
Not sure if we can document that in any ways, but the maximum number of advertising sets etc. are also dependent on the controller; so if not using the Zephyr controller, or if there's a dual-CPU board like the nRF5340, then the host Kconfig options may not reflect the actual number of advertising sets supported (but this is a general comment that also applies to e.g. number of connections etc.)